packages/core/src/scripts/tests/artifacts.test.ts

```diff
@@ -37,15 +37,17 @@ describe('postinstall', () => {
       'keystone.js': basicKeystoneConfig,
     });
     const recording = recordConsole({
-      'Would you like to update your Prisma and GraphQL schemas?': true,
+      'Replace the Prisma and GraphQL schemas?': true,
     });
     await runCommand(tmp, 'postinstall');
     const files = await getFiles(tmp, schemasMatch);
     expect(files).toEqual(await getFiles(`${__dirname}/fixtures/basic-project`, schemasMatch));
     expect(recording()).toMatchInlineSnapshot(`
       "Your Prisma and GraphQL schemas are not up to date
-      Prompt: Would you like to update your Prisma and GraphQL schemas? true
-      ✨ GraphQL and Prisma schemas are up to date"
+      Prompt: Replace the Prisma and GraphQL schemas? true
+      ✨ GraphQL and Prisma schemas are up to date
+      ✨ Building Keystone
+      ✨ Skipping Admin UI code generation"
```

Comment:
Let's remove "Building Keystone" and "Skipping..." as messages to be displayed.  Their verbosity is unhelpful.